src/aero_reps/aero_rep_modal_binned_mass.c

```diff
@@ -328,6 +328,55 @@ void aero_rep_modal_binned_mass_get_effective_radius__m(
   return;
 }
 
+/** \brief Get the effective particle surface area \f$r_{eff}\f$ (m)
+ *
+ * The surface area interface that exists between two specified phases within
+ * the modal/binned aerosol representation always returns zero. 
```

Comment:
```suggestion
 * The surface area interface that exists between two specified phases within
 * the modal/binned aerosol representation always returns zero, because no
 * specific internal structure is assumed for phases within modes or bins.
```